_events/2021/2021-12-dei-speaker-series.md

```diff
@@ -1,7 +1,6 @@
 ---
-title: Dr. Athina Frantzana of Spread the Word
-subtitle: US-RSE DEI-WG Speaker Series
-expires: 2021-12-11
+title: CANCELED - DEI Speaker Series December 10th Event
+expires: 2021-11-26
```

Comment:
I worry this might make the cancelation notice disappear fairly promptly? Maybe intentional.

Reply:
I don't have a great answer for "best practice" on this, but I'm open to suggestions. We'll be sending cancellation notices to those who registered, and the registration will be shut down, so I don't know how long we want this up.